src/constants/pets.js

```diff
@@ -1431,6 +1431,48 @@ module.exports = {
         }],
       },
     },
+    GRANDMA_WOLF: {
+      name: 'Grandma Wolf',
+      statModifiers: {
+        health: 1,
+        strength: 0.25,
+      },
+      abilityModifiers: {
+        0: {
+          mf1: [0.02, 0.02, 0.02, 0.02, 0.02],    // Math.floor((8 + this.level * 0.02) * 10) / 10
+          coins1: [0.02, 0.02, 0.02, 0.02, 0.02], // Math.floor((6 + this.level * 0.02) * 10) / 10
+          mf2: [0.02, 0.02, 0.02, 0.02, 0.02],    // Math.floor((4 + this.level * 0.02) * 10) / 10
+          exp: [0.01, 0.01, 0.01, 0.01, 0.01],    // Math.floor((3 + this.level * 0.02) * 10) / 10
+          mf3: [0.01, 0.01, 0.01, 0.01, 0.01],    // Math.floor((3 + this.level * 0.01) * 10) / 10
+          coins2: [0.01, 0.01, 0.01, 0.01, 0.01], // Math.floor((2 + this.level * 0.01) * 10) / 10
```

Comment:
You can use  a `descFn` to do the custom math for the values, like done here https://github.com/slothpixel/skyblock-parser/blob/master/src/constants/pets.js#L1043